composer.json

```diff
@@ -32,7 +32,7 @@
         "psr/log": "^1.0|^2.0|^3.0",
         "psr/simple-cache": "^1.0|^2.0|^3.0",
         "ramsey/uuid": "^4.2.2",
-        "symfony/console": "^6.0",
+        "symfony/console": "^6.0.3",
```

Comment:
I've bumped this as there is a bug in the previous versions where escaped strings were not parsed correctly, which was breaking some tests with `prefer-lowest`. I can revert this and remove those tests if needed.